src/chttpd/src/chttpd.erl

```diff
@@ -1036,6 +1036,8 @@ error_info({service_unavailable, Reason}) ->
     {503, <<"service unavailable">>, Reason};
 error_info({timeout, _Reason}) ->
     error_info(timeout);
+error_info({'EXIT', {Error, Stack}}) ->
+    error_info({Error, nil, Stack});
```

Comment:
this appears to just invoke a clause further down, which calls another. I think it better to shorten the path here. I think `error_info(Error)` ends up doing the same thing but calling error_info one further time not two?